common/selectors/selectAndDedupe.js

```diff
@@ -13,14 +13,12 @@ const dedupe = require("lib/dedupe");
 module.exports = function selectAndDedupe(group) {
   return group.reduce((result, options, index, arr) => {
     let current;
+    let sites = options.defaults ? options.sites.concat(options.defaults) : options.sites;
```

Comment:
It shouldn't exclude top sites as the default dupes are after the actual top sites via `concat`.